Dockerfile

```diff
@@ -13,14 +13,10 @@ RUN pip install -r requirements-dev.txt
 WORKDIR /build/
 ADD . /build/
 
-# The following command replaces the @VERSION@ string in setup.py and package.json
-# with the tagged version number from GIT_TAG or `0.0.0` if GIT_TAG is not set
-ARG VERSION=${GIT_TAG:-0.0.0}
-RUN echo "Version = $VERSION"
-# Update line 6 in setup.py
-RUN sed -i "6 s/0.0.0/$VERSION/" setup.py
+# The following command replaces the @VERSION@ string in setup.py with the tagged version number from GIT_TAG
+RUN sed -i s/@VERSION@/$GIT_TAG/ setup.py
 # Update line 3 in package.json
-RUN sed -i "3 s/0.0.0/$VERSION/" package.json
+RUN sed -i "3 s/0.0.0/${GIT_TAG:-0.0.0}/" package.json
```

Comment:
Can you drop the line number and instead search/replace the whole line (`"version": "0.0.0"`)?